packages/bot-web-ui/src/components/quick-strategy/form-wrappers/mobile-form-wrapper.tsx

```diff
@@ -0,0 +1,112 @@
+        if (run_panel.is_running) {
+            await setFieldValue('action', 'EDIT');
+            submitForm();
+            toggleStopBotDialog();
+        } else {
+            await setFieldValue('action', 'RUN');
+            submitForm();
+        }
+    };
+
+    return (
+        <div className='qs'>
+            <div className='qs__body'>
+                <div className='qs__body__content'>
+                    <ThemedScrollbars className='qs__form__container' autohide={false}>
+                        <div className='qs__body__content__title'>
+                            <div className='qs__body__content__description'>
+                                <Text size='xxs'>
+                                    {localize('Choose a template below and set your trade parameters.')}
+                                </Text>
+                            </div>
+                            <div className='qs__body__content__select'>
+                                <SelectNative
+                                    list_items={dropdown_list}
+                                    value={selected_strategy}
+                                    // label={localize(label)}
+                                    should_show_empty_option={false}
+                                    onChange={(e: React.ChangeEvent<HTMLSelectElement>) => {
+                                        onChangeStrategy(e.target.value);
+                                    }}
+                                />
+                            </div>
+                        </div>
+                        <div className='qs__body__content__description'>
+                            <div>
+                                <Text size='xxs'>{strategy.description}</Text>
+                            </div>
+                        </div>
+                        {/* <div className='qs__body__content__head'>
+                            <div className='qs__body__content__head__tabs'>
+                                {FORM_TABS.map(tab => {
+                                    const active = tab.value === active_tab;
+                                    const cs = 'qs__body__content__head__tabs__tab';
+                                    return (
+                                        <span
+                                            className={classNames(cs, { active, disabled: tab?.disabled })}
+                                            key={tab.value}
+                                            onClick={() => setActiveTab(tab.value)}
+                                        >
+                                            <Text size='xs' weight={active ? 'bold' : 'normal'}>
+                                                {tab.label}
+                                            </Text>
+                                        </span>
+                                    );
+                                })}
+                            </div>
+                        </div> */}
```

Comment:
Seems like this 2 content is identical for desktop and mobile. Do we have a way to make them reusable instead of duplicating them?

Reply:
@prince-deriv Initially it was one component but then the complexity and is_mobile checks started increasing so to keep it simple and straightforward I converted it into 2 dedicated components. And regarding the commented code, once the content team provides content we will uncomment the code and show the corresponding provided content, all will be done by config.